index.html

```diff
@@ -250,6 +250,76 @@ <h3 class="about-title">Skills</h3>
       </div>
     </section>
 
+    <!-- Get in touch -->
+
+    <section class="contact-form">
+      <div>
+        <h2 class="contact-header">
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </h2>
+      </div>
+
+      <footer class="footer">
+        <form action="#" method="post" class="form">
```

Comment:
- [ ] Good work so far, let's just make sure that Form uses the `action` attribute to integrate with [formspree service](https://formspree.io/html). this will allow for anyone to contact us through our portfolio :+1: 
* `action` value should be similar to `https://formspree.io/f/{form_id}`

Reply:
- [x] Added the action attribute with its id, thank you.